.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+name: CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
```

Comment:
let's keep running on 3.7 as well

Reply:
In the `Makefile`, 3.8 is hardcoded, Is there any way to make it dynamic?

Since if CI runs in `python 3.7` env, there is no `python 3.8` there and CI fails.

Reply:
you can set `PYTHON=python` in the env, that should use the default python